scio-hdfs/src/main/scala/com/spotify/scio/hdfs/package.scala

```diff
@@ -101,6 +101,15 @@ package object hdfs {
       self.context.makeFuture(HdfsTextTap(path))
     }
 
+    def saveAsHdfsTextFile(path: String, user: String): Future[Tap[String]] = {
```

Comment:
Instead of overloading the method with an extra argument, add it to the previous one with default `null`?